update/Pos/Update/MemState/Functions.hs

```diff
@@ -24,11 +25,11 @@ type UpdateVotes = HashMap PublicKey UpdateVote
 -- | Use a lock to perform operation on in-memory state of update system.
 --
 -- Currently we are using a single lock for everything, so this
--- function is just an alias for 'withBlkSemaphoreIgnoreTip'.
+-- function is just an alias for 'withStateLock'.
 withUSLock
-    :: (MonadReader ctx m, HasLens' ctx BlkSemaphore, MonadIO m, MonadMask m)
+    :: (MonadReader ctx m, HasLens' ctx StateLock, MonadIO m, MonadMask m)
     => m a -> m a
-withUSLock = withBlkSemaphore . const
+withUSLock = (withStateLockNoMetrics High) . const
```

Comment:
I think it should be `Low`, it's less important than block processing.

Reply:
Thanks, you're right.